blocks/banner/banner.css

```diff
@@ -0,0 +1,155 @@
+.banner-container .banner-wrapper {
+    max-width: unset;
+    padding-left: 15px;
+    padding-right: 15px;
+}
+
+.banner.block {
+    --banner-heading-font-family: 'Oswald', sans-serif;
```

Comment:
If fonts are used in more places, variables should be global, instead of block specific.